project4/gunrock_web/gunrock.cpp

```diff
@@ -167,7 +167,7 @@ int main(int argc, char *argv[]) {
   // The order that you push services dictates the search order
   // for path prefix matching
   services.push_back(new DistributedFileSystemService(DISKFILE));
-  services.push_back(new FileService(BASEDIR));
+  services.push_back(new FileService());
```

Comment:
I'd change the FileService back and hard code the `static` base directory here, at the FileService constructor.